sdk/ai/azure-ai-resources/azure/ai/resources/entities/connection_subtypes.py

```diff
@@ -316,20 +318,92 @@ def __init__(
         kwargs.pop("type", None)  # make sure we never somehow use wrong type
         super().__init__(target=target, type="git", credentials=credentials, **kwargs)
 
+class AzureBlobStoreConnection(BaseConnection):
+    """A Connection to an Azure Blob Datastore. Unlike other connection objects, this subclass has no credentials.
+    NOTE: This connection type is currently READ-ONLY via the LIST operation. Attempts to create or update
+    a connection of this type will result in an error.
+
+    :param name: Name of the connection.
+    :type name: str
+    :param target: The URL or ARM resource ID of the resource.
+    :type target: str
+    :param is_shared: For connections created for a project, this determines if the connection
+        is shared amongst other connections with that project's parent AI resource. Defaults to True.
+    :type is_shared: bool
+    :param tags: Tag dictionary. Tags can be added, removed, and updated.
+    :type tags: dict
+    """
+
+    def __init__(
+        self,
+        *,
+        target: str,
+        container_name: str,
+        account_name: str,
+        **kwargs,
+    ) -> None:
+        kwargs.pop("type", None)  # make sure we never somehow use wrong type
+        super().__init__(
+            target=target,
+            type="azure_blob",
+            container_name=container_name,
+            account_name=account_name,
+            **kwargs
+        )
+
+    @property
+    def container_name(self) -> str:
+        """The container name of the connection.
+
+        :return: the container name of the connection.
+        :rtype: str
+        """
+        if self._workspace_connection.tags is not None and CONNECTION_CONTAINER_NAME_KEY in self._workspace_connection.tags:
+            return self._workspace_connection.tags[CONNECTION_CONTAINER_NAME_KEY]
+        return ""
```

Comment:
None is prob a better default value. And then you can do something like `tags.get(CONNECTION_CONTAINER_NAME_KEY, None)`

Reply:
Oddly enough, returning None causes errors with mypy. I'll try messing around with them more and see if I can make mypy stop complaining about a None output.

Reply:
Ok got it.